Dockerfile

```diff
@@ -107,6 +117,7 @@ RUN \
   else \
     yarn install --frozen-lockfile; \
   fi
+USER root
```

Comment:
Question bc I don't understand it: why do we switch back to root here?

Reply:
I had assumed that eveything after has to run as root, up until the next `USER` - but @DrizzlyOwl should advise?

Reply:
We might be able to remove this line actually